app/templates/components/forms/reset-password-form.hbs

```diff
@@ -19,8 +19,24 @@
           <div class="field">
             {{#if token}}
               <div class="ui left icon input">
-                <i class="key icon"></i>
-                {{input type='password' name='password' placeholder=(t 'password') value=password}}
+                <i class="lock icon"></i>
+                  {{input type=(if showNewPass 'text' 'password') name='password' placeholder=(t 'Password') value=password}}
+                <div class="ui small basic icon buttons">
+                  <button type="button" class="ui button" id="eye2" {{ action 'showNewPassword'}}>
+                    <i class="{{if showNewPass 'hide' 'unhide'}} basic icon"></i>
+                  </button>
+                </div>
+              </div>
+              <br>
+              <br>
+              <div class="ui left icon input">
+                <i class="lock icon"></i>
+                  {{input type=(if showConfirmPass 'text' 'password') name='password_repeat' placeholder=(t 'Confirm Password') value=password_repeat}}
```

Comment:
There is no need for `value=password_repeat`, we don't have an actual field hence the identifier name will do for same.